src/pages/_id/civic/index.vue

```diff
@@ -147,7 +147,7 @@ export default {
         {
           hid: 'og:image',
           property: 'og:image',
-          content: 'https://liker.land/images/og/civic-v2.png',
+          content: `https://static.like.co/liker-og-image/${this.creator.user}`,
```

Comment:
Maybe refactor this line into a function?

Reply:
Updated, please check.